service_kit/utils/timer.py

```diff
@@ -0,0 +1,64 @@
+import time
+from logging import Logger
+from typing import Any, Callable
+
+from service_kit.logging import logger as skit_logger
+
+
+class Timer:
+    """
+    Just another timer utility that can be used as a context manager or a decorator.
```

Comment:
It's the only one in this library, though

```suggestion
    A utility for timing how long something took
```